youtube_dl/extractor/rai.py

```diff
@@ -410,6 +423,11 @@ def _real_extract(self, url):
                 ''' % self._UUID_RE,
                 webpage, 'content item id', default=None, group='id')
 
+        if not content_item_id:
+            content_item_id = self._search_regex(
+                r'/ContentItem-(?P<id>%s)\.html\?iframe' % self._UUID_RE,
```

Comment:
Add `<iframe` part to the regex and move this whole pattern to into previous `_search_regex` call.

Reply:
7b500a8a0269e5390d02ecfa55e57522def39ea4